### PR TITLE
Added disableEntityLink option to bar chart web component

### PR DESCRIPTION
**Pull request by dwnoble**
(UN ask)

Adds `disableEntityLink` option to bar chart web components to remove the entity link from x-axis:

9Yy4WfAEkGGD5PY

### CODE REVIEW

File: static/js/chart/draw_bar.ts [hunk 1]
```diff
@@ -317,7 +317,7 @@ export function drawStackBarChart(
     chartHeight,
     x,
     false,
-    labelToLink,
+    options?.disableEntityLink ? null : labelToLink,
```
Comment (keyurva) [inline static/js/chart/draw_bar.ts:320]: I'm not super familiar with this code but do we have tests for these functions / components? Would be good to add tests if we don't.

**Review by keyurva — APPROVED** [review]
Thanks Dan.

**Review by chejennifer — APPROVED** [review]
nice!

File: static/js/chart/draw_bar.ts (hunk 1 above)
Comment (dwnoble) [reply static/js/chart/draw_bar.ts:320]: We dont 😬  - testing web components doesnt work out of the box with our webdriver setup, and adding support has been on my todo list

**Pull request merged**